lte/gateway/c/core/oai/tasks/gtpv1-u/ebpf_dl_map.h

```diff
@@ -23,15 +23,18 @@
 struct bpf_map_val {
   uint32_t ip;
   uint32_t tei;
+  uint8_t imsi[16];
```

Comment:
Lets not add 3GPP terminology in eBPF, we can define this as binary blob that can be used in user-space to store data.

Reply:
lets keep it at the end of this struct and extend it to 64 bytes.

Reply:
rule number, version number, direction, etc.... I am pretty sure we will exhaust the 64 bytes, but if you want we can start at 16bytes.